lib/view/solve_crossword.dart

```diff
@@ -125,7 +126,8 @@ class TakePictureScreenState extends State<TakePictureScreen> {
     );
   }
 
-  Future<void> choosePhoto(BuildContext context) async {
+  // metody zwracaja tylko cropped image i pozniej solve wywolac osobno
```

Comment:
This comment can be misleading. Fix or delete it.

Reply:
gonna fix it